org.mlm.asteroids_revenge.yml

```diff
@@ -0,0 +1,30 @@
+app-id: org.mlm.asteroids_revenge
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: asteroids_revenge.x86_64
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+
+modules:
+  - name: AsteroidsRevenge
+    buildsystem: simple
+    build-commands:
+      - install -D asteroids_revenge.x86_64 /app/bin/asteroids_revenge.x86_64
```

Comment:
This won't work on aarch64, you are still building from it. Why not build from source?

Reply:
Building from source would would require godot 4, and its baseApp is only available upto 3.5 even though 4.2.1 is available at flatpak (without the baseApp)

Reply:
You need this then https://docs.flathub.org/docs/for-app-authors/maintenance#only-build-on-x86_64